src/lib/throttle.js

```diff
@@ -132,3 +136,55 @@ export function getReturnToken() : ?string {
         return token;
     }
 }
+
+type CustomThrottle = Throttle & {
+    isActive : () => boolean
+};
+
+export function buildFundingLogoThrottle(props : Object) : CustomThrottle {
+
+    let emptyThrottle = {
+        isActive:     () => false,
+        isEnabled:    () => false,
+        isDisabled:   () => true,
+        getTreatment: () => '',
+        log:          () => emptyThrottle,
+        logStart:     () => emptyThrottle,
+        logComplete:  () => emptyThrottle
+    };
+
+    let { layout, label, locale, browserLocale } = props;
+
+    if (browserLocale.country !== COUNTRY.US) {
```

Comment:
The `locale` prop is derived from browser locale. I don't think we need to check both separately. We can just rely on `locale`.

Reply:
I thought so too, but as I was poking around I saw that locale was passing in what 'locale' was set in the script by the user.  We need to check both what the user is setting the locale as, and what the browser is set at.

Reply:
Understood -- but if a locale is set by the merchant, checkout.js disregards the browser locale entirely, so there's a single source of truth. If we later enable this button universally for buyers in the US, that logic will preside. So it would be good to make the experiment match that.

Reply:
If locale is set by the merchant, for instance `locale: 'en_JP'`, browserLocale is still coming back as `{country: "US", lang: "en"}`.  The rest of checkout might disregard it, but the throttle seems to still be differentiating even when locale is set.

Reply:
Right -- `getBrowserLocale()` will always return the browser locale. And it may be different from the locale set by the merchant.

But the intention of the merchant set locale is to override the browser locale. All of the funding eligibility, content determination, even the locale which is passed to the hermes checkout flow, will be set to the locale set by the merchant. In every other scenario we normalize to a single locale. This experiment probably should not be an exception.